py4j-web/about.rst

```diff
@@ -1,3 +1,8 @@
+.. Marked as an orphan for Sphinx to suppress warnings. It's linked
+   by the custom theme.
+
+:orphan:
```

Comment:
Many thanks for this :-) I always ignored these warnings and did not look for a solution.